app/models/geometric-property.js

```diff
@@ -50,6 +51,51 @@ export const GEOMETRY_TYPES = [
   'rezoningArea',
 ];
 
+// returns true or false based on whether the change to the geometric
+// property was "meaningful"
+// first argument is a geometric-property model
+// todo: I think there are two separate handling here, one for
+// canonical geometries, one for regular
+export function isMeaningfulChange(geometricPropertyForType /* model */) {
+  // here, it gets set once by the constructor
+  // const initial = model.get(attribute);
+  const [
+    initial,
+    proposed, // upstream proposed should always be FC
+  ] = geometricPropertyForType.changedAttributes().proposedGeometry || [];
+
+  // "initial" means whatever is currently proposed. If it's _not_ empty and
```

Comment:
if `initial` is whatever is currently proposed, what is `proposed` ?

Reply:
@julialucyhogan it's poorly named - [changedAttributes](https://emberjs.com/api/ember-data/3.8/classes/DS.Model/methods/changedAttributes?anchor=changedAttributes) returns the old and new property when the model is dirty, meaning it isn't persisted back to the server. `initial` means what it was initially before something else was set on the attribute. See the dots for changedAttributes for better explanation